CADD.pm

```diff
@@ -213,13 +213,13 @@ sub run {
     $allele = $bvf->alt_alleles->[$ALT_NUM];
     $ref = $bvf->ref_allele_string;
 
-    if (($ALT_NUM + 1) == scalar(@{$bvf->alt_alleles})) {
+    if (($ALT_NUM + 1) >= scalar(@{$bvf->alt_alleles})) {
```

Comment:
It seems like the reason for the original issue is that multi-allelic variants are not handled properly and hence not annotated properly. Example to test: `chr1    83961   rs1191138855    GAAA    AAAA,G`